subtests/docker_cli/build/build.py

```diff
@@ -208,22 +209,28 @@ def parse_postprocess_commands(self, build_def):
         self.logdebug("Postprocessing commands: %s", commands)
         return result
 
-    @staticmethod
-    def basic_postprocess(build_def, command, parameter):
+    def basic_postprocess(self, build_def, command, parameter):
         del parameter  # not used
         if command == 'positive':
-            # Verify zero exit status + healthy output
-            if OutputGood(build_def.dockercmd.cmdresult,
-                          ignore_error=True):
+            # Verify zero exit status and healthy output
+            opg = OutputGood(build_def.dockercmd.cmdresult,
+                             ignore_error=True)
+            if opg:
                 return build_def.dockercmd.cmdresult.exit_status == 0
             else:
+                self.logwarn('Positive output expected but check failed: %s',
+                             str(opg))
                 return False
         elif command == 'negative':
-            # Verify non-zero exit status + healthy output
-            if OutputGood(build_def.dockercmd.cmdresult,
-                          ignore_error=True,
-                          skip=['error_check']):
+            # Verify non-zero exit status and no panics
+            opng = OutputNotBad(build_def.dockercmd.cmdresult,
```

Comment:
> Did you mean to make it nb ?

Uggg, yes...The number of letters has no impact on my ability to spell correctly.  Thankyou edautospellchecker 😀 

(really, I did mean ``opnb`` but my head is elsewhere)